web-app/HeimScripts/heatmap/limmaUtils.R

```diff
@@ -33,3 +29,11 @@ getDesign <- function(measurements) {
 getSubset1Length <- function(measurements) {
   sum(grepl(pattern = SUBSET1REGEX, x = colnames(measurements))) # returns number of column names satisfying regexp.
 }
+
+# to check if row  contains at least one row that contains 3 non missing values
+rowContainsAtLeastThreeData <- function (row) {sum(!is.na(row)) > 2}
+
```

Comment:
![screen shot 2016-02-04 at 13 34 00](https://cloud.githubusercontent.com/assets/2835281/12949936/e3f35c0a-d00a-11e5-9c2b-e195ce54c2c2.png)

It's actually valid